pkg/kn/commands/source/human_readable_flags.go

```diff
@@ -37,12 +37,21 @@ var sourceTypeDescription = map[string]string{
 	"KafkaSource": "Route events from Apache Kafka Server to addressable",
 }
 
+var sourceTypeBuiltin = map[string]string{
+	"ApiServerSource": "YES",
+	"SinkBinding":     "YES",
+	"PingSource":      "YES",
+	"ContainerSource": "YES",
+	"KafkaSource":     "NO",
+}
+
 // ListTypesHandlers handles printing human readable table for `kn source list-types`
 func ListTypesHandlers(h printers.PrintHandler) {
 	sourceTypesColumnDefinitions := []metav1beta1.TableColumnDefinition{
 		{Name: "Type", Type: "string", Description: "Kind / Type of the source type", Priority: 1},
 		{Name: "Name", Type: "string", Description: "Name of the source type", Priority: 1},
 		{Name: "Description", Type: "string", Description: "Description of the source type", Priority: 1},
+		{Name: "Built-In Source", Type: "string", Description: "Built-in source", Priority: 1},
```

Comment:
I'd shorten the name to `Built-In` due to the fact that we have only very short `YES, NO` values. 

And my preferred order would be `Type, Name, Built-In, Description` to allow more room for various description length.

Reply:
Thanks for the feedback @dsimansk 
I shortened it to `Built-In` and moved description to the end in this [commit](https://github.com/knative/client/pull/1246/commits/3e545e11342f91de0aa895456eb003e76bd248eb) 
Now the output looks like this:
```bash-5.1$ ./kn source list-types
TYPE              NAME                                   BUILT-IN   DESCRIPTION
ApiServerSource   apiserversources.sources.knative.dev   YES        Watch and send Kubernetes API events to addressable
ContainerSource   containersources.sources.knative.dev   YES        Generate events by Container image and send to addressable
PingSource        pingsources.sources.knative.dev        YES        Send periodically ping events to addressable
SinkBinding       sinkbindings.sources.knative.dev       YES        Binding for connecting a PodSpecable to addressable
```

I fixed the CLA issue by amending those commits with the correct email that I used to sign the CLA